app/assets/javascripts/application/messages.js

```diff
@@ -0,0 +1,96 @@
+var mumuki = mumuki || {};
+
+(function (mumuki) {
+
+    MessagesService = {
+        $body: function () {
+            return $('body')
+        },
+        $newMessageModal: function () {
+            return $('.new-message-modal');
+        },
+        $newMessageModalComponents: function () {
+            return MessagesService.$newMessageModal().find('.modal-body, .modal-footer');
```

Comment:
I know, sometimes `this` does not work, but this is not the case and it better reflects the intention.